lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -473,17 +475,11 @@ def test_grade_with_max_score_cache(self):
                 module_state_key=self.problem_location('p2')
             ).exists()
         )
-        location_to_cache = unicode(self.problem_location('p2'))
-        max_scores_cache = grades.MaxScoresCache.create_for_course(self.course)
 
-        # problem isn't in the cache
-        max_scores_cache.fetch_from_remote([location_to_cache])
-        self.assertIsNone(max_scores_cache.get(location_to_cache))
```

Comment:
You can assert that the course isn't cached before the first call, by calling `BlockStructureCache.get`.  And then assert that it is cached after the call.